Source/WebCore/platform/text/SegmentedString.h

```diff
@@ -27,7 +27,7 @@ namespace WebCore {
 
 // FIXME: This should not start with "k".
 // FIXME: This is a shared tokenizer concept, not a SegmentedString concept, but this is the only common header for now.
-constexpr LChar kEndOfFileMarker = 0;
+// constexpr LChar kEndOfFileMarker = 0;
```

Comment:
What’s going on here? If we should remove it please do, but don’t check it in commented out.